### PR TITLE
Add an endpoint to view a specific user's context

### PR DESCRIPTION
**Pull request by CelticMajora**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 node_modules/
+config.json
+package.json
```
Comment (CelticMajora) [inline .gitignore]: Not sure if these should be in the repo. Any thoughts @GreenMobius @amgassert @halawami ?

File: src/endpoints.js [hunk 2]
```diff
@@ -9,13 +9,17 @@ function messageHandler(message){
 
 	const args = message.content.slice(config.prefix.length).trim().split(/ +/g);
 	const command = args.shift().toLowerCase();
-	const uid = message.user.id;
+	//const uid = message.user.id;
```
Comment (CelticMajora) [inline src/endpoints.js]: This line was breaking the whole thing, but it appears to be being passed to democracy. What exactly is democracy expecting? Should it be author.id?

Comment (GreenMobius) [reply src/endpoints.js]: What I said right before I left the other day was the issue I was getting with this line- Democracy doesn't need it.

File: .gitignore (hunk 1 above)
Comment (GreenMobius) [reply .gitignore]: the config.json should have been fixed in mine unless it was re-added.
package and package-lock detail the dependencies, which might be useful to keep in. Worst-case they're a few kilobytes.

File: src/endpoints.js (hunk 2 above)
Comment (GreenMobius) [reply src/endpoints.js]: propose just removing it

**Comment by CelticMajora** [comment]
> Keep package and package lock, remove commented line (both detailed in comments).
> 
> Approved with changes

Just went through and made the proposed changes

**Pull request merged**